src/dmd/backend/dwarf.d

```diff
@@ -16,7 +16,7 @@ extern (C++):
 
 nothrow:
 
-enum DWARF_VERSION = 3;
+immutable ubyte[] DWARF_SUPPORTED_VERSIONS = [3, 4, ];
```

Comment:
It's way over-engineered to have an array for 2 values.

Reply:
Agreed, version 5 supports mostly everything that version 3 and 4 support, so there's no need to limit the versions given beyond `< 2` (due to `DW_LANG_D` not existing until dwarf v3).